tools/autograd/gen_python_functions.py

```diff
@@ -702,6 +706,26 @@ def process_function(name, declarations):
             typedef, next_index = emit_namedtuple_return_type_def(declaration, next_index)
             env['declare_namedtuple_return_types'] += typedef
 
+        # In-place operator, should be method only. Deprecate it.
+        if not is_module and not has_self and name.endswith('_') and not name.startswith('_'):
+            deprecated_functions = ['abs_', 'acos_', 'addmv_', 'alpha_dropout_', 'as_strided_', 'asin_', 'atan_',
```

Comment:
ya, I don't think putting a big list here is a good idea.

For example, I don't see how we can deprecate a bunch of these -- our nn functional code calls a bunch of these, even if they aren't in the "correct" nn namespace (check all the dropouts, for example).

Reply:
I believe that the use of all of these function variants have been eliminated, at least in PyTorch codebase (No compilation error if we simply remove them from the scene).

Where would you like to put the list? This list is not supposed to grow anyway and we only hide them here temporarily.

What is your suggesetion for handling misplaced nn functions?

Reply:
> I believe that the use of all of these function variants have been eliminated, at least in PyTorch codebase (No compilation error if we simply remove them from the scene).

What about something like this?
https://github.com/pytorch/pytorch/blob/eb76b7a564121c5fede749ad7d0a36f2b61a0a95/torch/nn/functional.py#L800

Reply:
OK, fair enough. I reverted all changes of all nn functions that are misplaced in `torch.` (in this context, `*dropout` functions). We can probably decide what we should do with those functions in the future. Could you review again?

Reply:
Do you in general treat `relu_` as an nn function? If so, I'll also remove its deprecation as well.

Reply:
@gchanan This PR should be ready now. Feel free to review. Thanks!